test/recipe/admin/pages/specs/report-page.vue

```diff
@@ -45,6 +45,36 @@ const variants = [
       ]
     })
   },
+  {
+    name: "Show When",
+    descriptor: giveMe.aReportPage({
+      data: {
+        showTab2: false
+      },
+      tabs: [
+        giveMe.aReportPageTab({
+          id: "tab-1",
+          title: "Spec: Tab 1",
+          icon: giveMe.anExpected({ testId: "icon 1", value: "I." }),
```

Comment:
should include showWhen for both tab and tab content. and add a page context behavior to input component to add a desired key to the context so that tab and tab content gets hidden/shown when input is changed